daemon/delete.go

```diff
@@ -25,7 +26,7 @@ func (daemon *Daemon) ContainerRm(name string, config *ContainerRmConfig) error
 			// TODO: why was just job.Error(err) without return if the function cannot continue w/o container name?
 			//job.Error(err)
 		}
-		parent, n := path.Split(name)
+		parent, n := filepath.Split(name)
```

Comment:
Hmm, I don't think we want filepath here since it's not a filepath, it's a container name that may include forward slashes.